config/metadata/rulesMeta.yaml

```diff
@@ -495,24 +506,24 @@ groups:
               - EMAThroughputSampler
               - WindowedThroughputSampler
               - TotalThroughputSampler
-        summary: is the dynamic sampler to use if the rule matches.
+        summary: is the Dynamic Sampler to use if the rule matches.
         description: >
           The sampler to use if the rule matches. If this is set, the sample
           rate will be determined by this downstream sampler. If this is not set,
-          the sample rate will be determined by the Drop flag or the SampleRate
+          the sample rate will be determined by the `Drop` flag or the `SampleRate`
           field.
       - name: Drop
         type: bool
         summary: indicates whether to drop the trace.
         description: >
-          Indicates whether to drop the trace if it matches this rule. If true,
-          the trace will be dropped. If false, the trace will be kept.
+          Indicates whether to drop the trace if it matches this rule. If `true`,
+          the trace will be dropped. If `false`, the trace will be kept.
       - name: SampleRate
         type: int
         summary: is the sample rate to use.
         description: >
           If the rule is matched, there is no Sampler specified, and the Drop
```

Comment:
@kentquirk For this description, do the previous three conditions (rule is matched, no specified Sampler, and `Drop` flag is `false`) have to be met before `SampleRate` here is used?

Reply:
That's the point of the note above -- one of 3 things happens, in the specified order.

Reply:
So all three conditions have to occur to use this Sample Rate. Okay, thank you for clarifying! I was having difficulty parsing the sentence and wanted to be sure.